### PR TITLE
reinstalled active admin

### PR DESCRIPTION
**Pull request by PaulStorberg**

### CODE REVIEW

File: app/admin/users.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ActiveAdmin.register User do
-permit_params :email, :first_name, :id, :last_name, :middle_name, :password, :password_confirmation
+  permit_params :email, :password, :password_confirmation, :first_name, :middle_name, :last_name
```
Comment (kevinstorberg) [inline app/admin/users.rb]: @PaulStorberg Alphabetize this

File: app/models/user.rb [hunk 2]
```diff
@@ -1,8 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :trackable, :validatable
   include SnoopDogg
```
Comment (kevinstorberg) [inline app/models/user.rb]: @PaulStorberg Put included concerns above everything else, directly below the class name and without space between the class names and the concern

File: app/models/user.rb [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
   rolify
-  devise :database_authenticatable, :registerable,
+  devise :database_authenticatable,
```
Comment (kevinstorberg) [inline app/models/user.rb]: @PaulStorberg Put these one one line. They are not many of them. They fit on one line.

**Review by kevinstorberg — CHANGES_REQUESTED** [review]

File: app/admin/users.rb (hunk 1 above)
Comment (PaulStorberg) [reply app/admin/users.rb]: complete

File: app/models/user.rb (hunk 2 above)
Comment (PaulStorberg) [reply app/models/user.rb]: complete

File: app/models/user.rb (hunk 3 above)
Comment (PaulStorberg) [reply app/models/user.rb]: complete

**Pull request merged**